README.md

```diff
@@ -96,7 +96,7 @@ repo](https://github.com/rapidsai/notebooks-contrib).
 | **Linear Models for Regression or Classification** | Linear Regression (OLS) | Multi-node multi-GPU via Dask |
 | | Linear Regression with Lasso or Ridge Regularization | Multi-node multi-GPU via Dask |
 | | ElasticNet Regression | |
-| | Logistic Regression | |
+| | Logistic Regression | Multi-node multi-GPU via Dask-GLM [demo](https://github.com/daxiongshu/rapids-demos) |
```

Comment:
Do you have any plans to push this notebook & readme to a RAPIDS repository? It would also be useful for users to provide a link to the Dask-GLM logistic regression docs / repo.

Reply:
I was trying to push the notebook to `notebooks-contrib` but it seems that repo is under some changes so @JohnZed suggested to link my repo for now.

I'll add the link to `dask-glm` and documents in the readme of my repo. Thank you!

Reply:
Yes, I support leaving it in a personal repo with a readme now until reorganizations in notebooks-contrib etc. are finalized so we don't have to keep moving around the link.